### DISCUSSION TITLE
RabbitMQ.Client.OAuth2 2.0.0 GA is not available on Nuget

### DISCUSSION THREAD

**Discussion by Lyphion** [rabbitmq/rabbitmq-dotnet-client#1726] [General]
### Describe the bug

The RabbitMQ.Client.OAuth2 Nuget package is missing the newest changes. Currently only Version 1.0.0 and 2.0.0-rc.1 are available.

The docs were updated with new features and functions. Maybe in this context a note in the docs on using an Issuer instead of the Token Endpoint is fitting.

### Reproduction steps

1. Search for RabbitMQ.Client.OAuth2 on Nuget.org
2. Checking out newest release
3. See that features are missing

### Expected behavior

New version with changes are published/available

**Comment by michaelklishin** [comment]
@Lyphion what specifically leads you to believe that `RabbitMQ.Client.OAuth2` 2.0.0 GA is supposed to be out? The fact that it is not yet release is not a "bug".

**Reply by Lyphion** [reply]
I didn't mean 2.0.0 GA directly. I thought a new RC would be created after some time or with the big 7.0 update all parts get a new version to verify compatibility.

**Reply by Lyphion** [reply]
And yes, this is not really a bug (except of the usage of the new version in the docs which is currently not published). But I thought a feature request was not so fitting,

**Reply by michaelklishin** [reply]
@Lyphion GitHub Discussions have been around for a bit over three years now. Consider using them instead of filing issues when you have a question.

**Comment by lukebakken** [comment]
Thanks for noticing. I will check, but I don't think there's anything preventing a `2.0.0` GA release.

https://github.com/rabbitmq/rabbitmq-dotnet-client/issues/1727

**Comment by lukebakken** [comment]
Version 2 is available - https://www.nuget.org/packages/RabbitMQ.Client.OAuth2/
